README.md

```diff
@@ -51,6 +51,26 @@ const stripe = Stripe('sk_test_...');
 On older versions of Node, you can use [promises](#using-promises)
 or [callbacks](#using-callbacks) instead of `async`/`await`.
 
+## Initialize with config object
+
+The package can be initialized with several options:
+
+```js
+import ProxyAgent from 'https-proxy-agent';
+
+const stripe = Stripe('sk_test_...', {
+  apiVersion: '2019-08-08', // Specify an API version
+  maxNetworkRetries: 1, // The amount of times a request should be retried
+  httpAgent: new ProxyAgent(process.env.http_proxy), // Set a proxy agent
+  timeout: 1000, // Amount of time in ms before a request times out
+  host: 'api.example.com', // Host that requests are made to
```

Comment:
Wasn't sure on whether to add `host` and `port` to this example, since you really should only be changing these if you know what you are doing. Thoughts?

Reply:
I agree with your logic here; if someone is doing that, they'll (hopefully) just dig into the details to see how.

🤔 Maybe make mention of them, but not actually include them?